src/components/SendModalComponent.vue

```diff
@@ -42,6 +45,21 @@ onMounted(async () => {
 });
 
 async function sendAsset() {
+  // Check network and update previous values
+  let chainID = await web3.eth.getChainId();
+  if (chainID === 22) {
```

Comment:
Add 22 in a variable and you can put this in a global conf file:

```
const CHAIN_IDS = {
  L14: 22,
  L16: 2828
}
```

Because then it gives more sense / context to that number.

and then:

```js
if (chainId === CHAIN_IDS.L14)
```

When we read we see "ah ok, if chain Id equals the chain id of Lukso L14"